src/systems/systems/src/pneumatic/mod.rs

```diff
@@ -335,11 +466,12 @@ impl Precooler {
                 - container_one.temperature().get::<degree_celsius>(),
         );
 
-        let temperature_change =
-            temperature_gradient * (1. - (-self.coefficient * context.delta_as_secs_f64()).exp());
+        let temperature_change = temperature_gradient / Self::HEAT_CAPACITY_CONSTANT_PRESSURE
+            * (self.heat_transfer_coefficient * context.delta_as_secs_f64());
```

Comment:
Hmm, this is really an energy to transfer. I think this is a good idea, but we should name the variable appropriately.

Reply:
I know. The thing is that it's neither energy nor a temperature but something in between. I try to find another variable name.

Reply:
I changed the name of the variable